frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.jsx

```diff
@@ -131,6 +138,10 @@ function Version({ version }) {
 }
 
 function HostingCTA() {
+  if (MetabaseSettings.isEnterprise()) {
```

Comment:
It looks like there are some redundant checks left

With early return when the instance is hosted, we can just add
```
// line: 29
const shouldShowHostedCta = !MetabaseSettings.isEnterprise()

// lines 36 and 80
{shouldShowHostedCta && <HostingCTA />}
```
What do you think?

Reply:
@alxnddr Yes, totally. Changed. Thank you very much for pointing it out. 🙇